assets/section-main-product.css

```diff
@@ -770,8 +770,8 @@ a.product__text {
 }
 
 @media screen and (max-width: 749px) {
-  .product--thumbnail .is-active > .product__modal-opener:not(.product__modal-opener--image),
-  .product--thumbnail_slider .is-active > .product__modal-opener:not(.product__modal-opener--image) {
+  .product--thumbnail .is-active .product__modal-opener:not(.product__modal-opener--image),
+  .product--thumbnail_slider .is-active .product__modal-opener:not(.product__modal-opener--image) {
     display: none;
   }
 
```

Comment:
The modal opener is still a descendant of `is-active`

https://screenshot.click/24-22-7aepp-g3r0n.png

The issue here seems to be of specificity:

https://screenshot.click/24-16-zdgsv-wac5e.png

By adding the `.product-media-container` class, it also works:

https://screenshot.click/24-20-b1m2o-oyqyn.png